code/config/lib/Loot/Config/CLI.hs

```diff
@@ -0,0 +1,89 @@
+{- This Source Code Form is subject to the terms of the Mozilla Public
+ - License, v. 2.0. If a copy of the MPL was not distributed with this
+ - file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ -}
```

Comment:
If you intend to add license information to each file, please replace this boilerplate with machine-readable SPDX identifier metadata:

- https://spdx.org/spdx-specification-21-web-version#h.twlc0ztnng3b (spec)
- https://lwn.net/Articles/739183/ (article)
- https://github.com/torvalds/linux/blob/6f0d349d922ba44e4348a17a78ea51b7135965b1/arch/mips/boot/compressed/uart-alchemy.c#L1 (usage example)

In this case it should be:

```
{- SPDX-License-Identifier: Apache-2.0 -}
```